### DISCUSSION TITLE
How can i clean the github search history?

### DISCUSSION THREAD

**Discussion by bhaweshchaudhary** [community/community#112913] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

When we click on github search bar, it presents us with repos that we have visited in the past or have searched. How to remove them ?

**Comment by charan-happy** [comment]
Hi @bhaweshchaudhary ,  Please go through below path

Go to your profile >> settings >> search section >> Under the "Search" section, you'll see an option labeled "Clear search history." Click on this button. >> finally GitHub asks you to confirm.

That's it.

**Comment by Trkzi-Omar** [comment]
The above solution didn't work for me.
GitHub does not provide a built-in feature to clear your search history. However, you can clear your browser's cache (I found it under `localStorage`) to remove your search history on GitHub. 

Here are the general steps:

1. Open your browser settings.
2. Navigate to the privacy or history section.
3. Select the option to clear browsing data (or find the specific entry for `search_history`).
4. Choose to clear cached files (or the specific entry in `localStorage`).

**Comment by rahulbatra-19** [comment]
GitHub saves your recently visited repositories in your browser’s localStorage, under a key called **jump_to:page_views**. To remove them:

### Steps (Chrome example, similar for other browsers):
1. Open GitHub in your browser.
2. Right–click anywhere on the page → Inspect (or press Ctrl+Shift+I / Cmd+Option+I).
3. In DevTools, go to the Application tab.
4. In the left sidebar, expand Local Storage → select https://github.com.
5. In the search bar above the storage table, type jump_to:page_views.
6. Select that entry and click the ❌ (delete) icon.
